src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -487,6 +533,17 @@ private void SendStreamingMessage(StreamingMessage msg)
             _eventManager.Publish(new OutboundEvent(_workerId, msg));
         }
 
+        private void PublishRpcChannelReadyAndRegisterFunction()
+        {
```

Comment:
Nit: Register functions does not happen for any language worker that is started at application level i.e. placeholder channel.

Reply:
Getting the channel ready and registering for functions are two separate actions. This needs to be refactored. It is outside scope of this PR. I will address this later